src/app/app-platform/ContentApp.h

```diff
@@ -122,6 +143,7 @@ class DLL_EXPORT ContentApp
 
 protected:
     EndpointId mEndpointId = 0;
+    std::vector<SupportedCluster> mSupportedClusters;
```

Comment:
We should probably make the default be something that makes sense. Or make changes in AppImpl.h since its passing an empty list for the dummy content apps.  

For example, there are mandatory clusters on a content app - KeypadInput, AppLauncher, AppBasic. Then there are the clusters which the code assumes are present (see https://github.com/project-chip/connectedhomeip/blob/bd0422b25ac329323b0ba93808fe7084ec8f2c0a/examples/tv-app/tv-common/src/AppTv.cpp#L315)

Reply:
Interesting. I have some follow up questions:

* Is this only really relevant for the Linux/native TvApp? For the Android TvApp I would assume we (only) want to populate the ContentApp clusters list from the Manifest (?). If so I don't think it would be the best to add defaults to this mSupportedClusters list (as it might conflict/overlap with the manifest declarations on Android. Or is it not an expectation for the manifest to declare all clusters, just what is extra beyond what is deemed default?
* Is the list in [AppImpl.h](https://github.com/project-chip/connectedhomeip/blob/e985c267b22cbb6d207b9c145ddd77a57fb7730c/examples/tv-app/linux/AppImpl.h#L137) only used for Linux testing?
* Is the static/default cluster/attribute list declared in AppTv.cpp that you linked available to the native ContentAppPlatform in some way? How/where is it used?

Reply:
After talking to @lazarkov about this I have added a list of default supported cluster identifiers to the mContentApp list initializer in AppImpl (for Android) and AppTv (for Linux). These lists are only used for testing to add sensible defaults and pass the new AccountLogin check.